google/cloud/aiplatform/prediction/local_model.py

```diff
@@ -522,6 +525,81 @@ def deploy_to_local_endpoint(
         finally:
             pass
 
+    def deploy_to_local_endpoint_lastingly(
```

Comment:
This method shouldn't be needed. Users should call `local_model.deploy_to_local_endpoint()` to get an endpoint object, then `.serve()` on that object to start the container.